templates/includes/side_navigation.html

```diff
@@ -0,0 +1,10 @@
+<a href="javascript:void(0)" class="open-close-btn close-btn" onclick="closeNav()">
```

Comment:
Das war ja bereits so, aber da wir eh jquery verwenden könnte man das bequem auch direkt in der funktion lösen und bräuchten kein inline javascript.

Reply:
@busykoala Könntest du das bitte entsprechend anpassen?

Reply:
Ja klar. Aber dann mach am besten ein jira draus und markiere es hier als erledigt...